index.json

```
@@ -78,5 +78,246 @@
+        "cAnswer": "Source/ArrayRelatedOperations/sum_range_array.c",
+        "jsAnswer": "Source/ArrayRelatedOperations/sum_range_array.js",
+        "enAlususAnswer": "Source/ArrayRelatedOperations/sum_range_array.en.alusus",
+        "arAlususAnswer": "Source/ArrayRelatedOperations/sum_range_array.ar.alusus"
+    },
+    {
+        "enPrompt": "Find the position of a given element in the array starting from its end",
+        "arPrompt": "أوجد موقع عنصر معطى في مصفوفة بدءاً من نهايتها",
+        "cAnswer": "Source/ArrayRelatedOperations/rfind.c",
+        "jsAnswer": "Source/ArrayRelatedOperations/rfind.js",
+        "enAlususAnswer": "Source/ArrayRelatedOperations/rfind.en.alusus",
+        "arAlususAnswer": "Source/ArrayRelatedOperations/rfind.ar.alusus"
+    },
+    {
+        "enPrompt": "Insert an element at a given position in the array",
+        "arPrompt": "احشر عنصر في موقع معطى ضمن المصفوفة",
+        "cAnswer": "Source/ArrayRelatedOperations/insert.c",
+        "jsAnswer": "Source/ArrayRelatedOperations/insert.js",
+        "enAlususAnswer": "Source/ArrayRelatedOperations/insert.en.alusus",
+        "arAlususAnswer": "Source/ArrayRelatedOperations/insert.ar.alusus"
+    },
+    {
+        "enPrompt": "Delete an element at a given position in the array",
+        "arPrompt": "احذف عنصر من موقع معطى ضمن المصفوفة",
+        "cAnswer": "Source/ArrayRelatedOperations/erase.c",
+        "jsAnswer": "Source/ArrayRelatedOperations/erase.js",
+        "enAlususAnswer": "Source/ArrayRelatedOperations/erase.en.alusus",
+        "arAlususAnswer": "Source/ArrayRelatedOperations/erase.ar.alusus"
+    },
+    {
+        "enPrompt": "Write Floyd Warshall algorithm",
+        "arPrompt": "اكتب خوارزمية Floyd Warshall",
+        "cAnswer": "Source/GraphAlgorithms/floyd_warshall.c",
+        "jsAnswer": "Source/GraphAlgorithms/floyd_warshall.js",
+        "enAlususAnswer": "Source/GraphAlgorithms/floyd_warshall.en.alusus",
+        "arAlususAnswer": "Source/GraphAlgorithms/floyd_warshall.ar.alusus"
+    },
+    {
+        "enPrompt": "check if a character is from the alphabet",
+        "arPrompt": "اختبر فيما إذا كان محرف من الأبجدية",
+        "cAnswer": "Source/StringRelatedOperations/is_alpha.c",
+        "jsAnswer": "Source/StringRelatedOperations/is_alpha.js",
+        "enAlususAnswer": "Source/StringRelatedOperations/is_alpha.en.alusus",
+        "arAlususAnswer": "Source/StringRelatedOperations/is_alpha.ar.alusus"
+    },
+    {
+        "enPrompt": "check if a character is a digit",
+        "arPrompt": "اختبر فيما إذا كان محرف هو رقم بين 0 و 9",
+        "cAnswer": "Source/StringRelatedOperations/is_digit.c",
+        "jsAnswer": "Source/StringRelatedOperations/is_digit.js",
+        "enAlususAnswer": "Source/StringRelatedOperations/is_digit.en.alusus",
+        "arAlususAnswer": "Source/StringRelatedOperations/is_digit.ar.alusus"
+    },
+    {
+        "enPrompt": "check if a string is alphanumeric",
```

```suggestion
        "enPrompt": "check if all characters of a string are alphanumeric",
```

كي يكون الوصف مماثلا للنسخة العربية.